content/using_ec2_spot_instances_with_eks/scaling/test_ca.md

```diff
@@ -88,15 +92,15 @@ You should also be able to visualize the scaling action using kube-ops-view. Kub
 ![Scaling up to 10 replicas](/images/using_ec2_spot_instances_with_eks/scaling/scaling-kov-10-replicas.png)
 
 {{% notice info %}}
-Given we started from 0 capacity in both Ec2Spot nodegroups, this should trigger a scaling event for Cluster Autoscaler. Can you predict which size (and type!) of node will be provided ? 
+Given we started from 2 node capacity in both Spot node groups, this should trigger a scaling event for Cluster Autoscaler. Can you predict which size (and type!) of node will be provided ? 
 {{% /notice %}}
 
 #### Challenge
 
 Try to answer the following questions:
 
- - Could you predict what should happen if we increase the number of replicas to 13 ? 
- - How would you scale up the replicas to 13 ? 
+ - Could you predict what should happen if we increase the number of replicas to 23 ? 
+ - How would you scale up the replicas to 23 ? 
  - If you are expecting a new node, which size will it be: (a) 4vCPU's 16GB RAM or (b) 8vCPU's 32GB RAM ? 
  - Which AWS instance type you would expect to be selected ?
```

Comment:
Which AWS instance type

EC2 instance type

Reply:
As Ran pointed out line #105, lets change **AWS Instance type** to **EC2 Instance type**